include/kodlab_mjbots_sdk/imu_data.h

```diff
@@ -95,6 +95,16 @@ class IMUData
    */
   Eigen::Quaternion<Scalar> world_offset_ = {1, 0, 0, 0};
 
+  /**
+   * @brief Constant for taking degrees to radians
+   */
+  static constexpr float kDegsToRads_ = M_PI/180.0;
+  
+  /**
+   * @brief Constant for taking degrees to radians
+   */
+  static constexpr float kRadsToDegs_ = 180.0/M_PI;
```

Comment:
The correct way to handle these constants (if the definitions and implementation were separated into header and source files) would be with an `extern` definition ([see this post](https://stackoverflow.com/a/2043667)).  This is outside the scope of this PR, but it might be worth separating the definition and implementation of `IMUData` into separate files and using [explicit template specialization](https://isocpp.org/wiki/faq/templates#separate-template-fn-defn-from-decl).  Then modifying this constant definition to use `extern`.

Reply:
Just to be clear, this is not an action item for now, right?